test/unit/mdl-menu/mdl-simple-menu.test.js

```diff
@@ -52,10 +56,24 @@ test('get/set open', t => {
   t.end();
 });
 
-test('items returns the container element for the menu items', t => {
+test('constructor throws if no role on menu items', t => {
+  t.throws(() => new MDLSimpleMenu(bel`
+    <div class="mdl-simple-menu"><ul class="mdl-simple-menu__items"></ul></div>
+  `));
+  t.end();
+});
+
+test('constructor throws if role on menu is invalid (not menu or listbox)', t => {
+  t.throws(() => new MDLSimpleMenu(bel`
+    <div class="mdl-simple-menu"><ul class="mdl-simple-menu__items" role="dialog"></ul></div>
+  `));
+  t.end();
+});
+
+test('items returns all menu items', t => {
   const {root, component} = setupTest();
-  const items = root.querySelector(strings.ITEMS_SELECTOR);
-  t.equal(component.items, items);
+  const items = [...root.querySelectorAll('[role="menuitem"]')];
```

Comment:
Yeah so it looks like this is because Chrome on Android doesn't support `Array.from` http://kangax.github.io/compat-table/es6/#test-Array_static_methods we may want to consider using something like `babel-transform-runtime` if we keep running into things like these but I'm hesitant to include it just yet.